tests/conftest.py

```diff
@@ -39,6 +63,25 @@ def docker(event_loop):
 
 
 @pytest.fixture
+def swarm(event_loop, docker):
```

Comment:
> It makes all sense imho: a Docker in Swarm mode is something very different to handle than a simple Docker daemon:

Not really a Docker daemon is still a docker daemon, being a part of Swarm or not.
You can still run containers and do all the other operations that you would do outside of a swarm.
The access to the 'services' api depends, if you are not a master node for example you cannot create services, so part of the APIs don't work.

> If you now run the tests on your own Docker instance, if the daemon is already in swarm mode, it will not work. I can imagine that people who run tests doesn't want the test to remove all their services.

This is something we need to decide after we removed docker compose and the dind because we can dirty the local env during testing.